x-pack/test/security_solution_cypress/cypress/screens/inspect.ts

```diff
@@ -90,12 +88,12 @@ export const INSPECT_BUTTONS_IN_SECURITY: InspectButtonMetadata[] = [
         altInspectId: '[data-test-subj="events-viewer-panel"]',
         id: EVENT_CONTAINER_TABLE_NOT_LOADING,
       },
-      {
-        title: 'Host risk',
-        tab: RISK_DETAILS_NAV,
-        customIndexPattern: 'ml_host_risk_score_latest_default',
-        id: HOST_BY_RISK_TABLE,
-      },
+      // {
```

Comment:
@machadoum same question here about reliably enabling risk score for the associated test